### PR TITLE
Fix OVF import failure when location restrictions are on

### PR DESCRIPTION
**Pull request by zoran15**
Fix OVF import failure when location restrictions are on due to images being created in multi-regions. Since this change is internal to the importer, added releaseTrack flag to indicate which Compute API release track is allowed to be used. Image location is only in Beta and we shouldn't use Beta API in GA release.

### CODE REVIEW

File: cli_tools/common/utils/daisy/resource_labeler_test.go [hunk 1]
```diff
@@ -139,11 +164,51 @@ func TestUpdateWorkflowImagesLabelled(t *testing.T) {
...
+
+	rl := createTestResourceLabeler(buildID, userLabels)
+	rl.ImageLocation = "europe-west5"
+
+	rl.LabelResources(w)
+
+	validateLabels(t, &(*w.Steps["cimg"].CreateImages).ImagesBeta[0].Image.Labels, "gce-image-import",
+		buildID, &existingLabels)
+
+	assert.Equal(t, "europe-west5", (*w.Steps["cimg"].CreateImages).ImagesBeta[0].Image.StorageLocations[0])
```
Comment (EricEdens) [inline cli_tools/common/utils/daisy/resource_labeler_test.go]: Should the test fail if there are other labels in the list? Does the test not care?

File: cli_tools/gce_ovf_import/main.go [hunk 2]
```diff
@@ -59,6 +60,7 @@ var (
 	gcsLogsDisabled             = flag.Bool("disable-gcs-logging", false, "do not stream logs to GCS")
 	cloudLogsDisabled           = flag.Bool("disable-cloud-logging", false, "do not stream logs to Cloud Logging")
 	stdoutLogsDisabled          = flag.Bool("disable-stdout-logging", false, "do not display individual workflow logs on stdout")
+	releaseTrack                = flag.String("release-track", ovfimporter.GA, fmt.Sprintf("Release track of OVF import. One of: %s, %s or %s. Impacts which compute API release track is used by the import tool.", ovfimporter.Alpha, ovfimporter.Beta, ovfimporter.GA))
```
Comment (EricEdens) [inline cli_tools/gce_ovf_import/main.go]: Is there an idiom for validating that the user actually passed ga, alpha, or beta?

File: cli_tools/gce_ovf_import/ovf_importer/ovf_importer_test.go [hunk 3]
```diff
@@ -182,6 +183,50 @@ func TestSetUpWorkflowHappyPathFromOVAExistingScratchBucketProjectZoneAsFlags(t
 	assert.Equal(t, "gs://bucket/folder/ovf-import-build123/ovf/", oi.gcsPathToClean)
 }
 
+func TestSetUpWorkflowUsesImageLocationForBetaReleaseTrack(t *testing.T) {
```
Comment (EricEdens) [inline cli_tools/gce_ovf_import/ovf_importer/ovf_importer_test.go]: How about for GA? Do you want to assert that the location is not set?

File: cli_tools/common/utils/daisy/resource_labeler_test.go (hunk 1 above)
Comment (zoran15) [reply cli_tools/common/utils/daisy/resource_labeler_test.go]: That's already checked using existingLabels

Comment (EricEdens) [reply cli_tools/common/utils/daisy/resource_labeler_test.go]: The 'ImagesBeta[0].Image.StorageLocations' list, I mean. For example, to ensure that a duplicate doesn't sneak in there or something like that. A quick check would be to assert that the list has length one, I'd think.

Comment (zoran15) [reply cli_tools/common/utils/daisy/resource_labeler_test.go]: Added length==1 assertion

File: cli_tools/gce_ovf_import/main.go (hunk 2 above)
Comment (zoran15) [reply cli_tools/gce_ovf_import/main.go]: Not sure it's an idiom, but I added a validation function for release-track flag.

File: cli_tools/gce_ovf_import/ovf_importer/ovf_importer_test.go (hunk 3 above)
Comment (zoran15) [reply cli_tools/gce_ovf_import/ovf_importer/ovf_importer_test.go]: Added a test case for GA

**Pull request merged**